draft-packages/modal/KaizenDraft/Modal/Presets/matchMedia.mock

```diff
@@ -0,0 +1,15 @@
+// Must be imported before the tested file
```

Comment:
I wonder if this should be a global mock given it's used in every package (that has tests) that uses Illustrations. 🤔